dudetronics.nostar.DroidStar.yml

```diff
@@ -0,0 +1,48 @@
+app-id: dudetronics.nostar.DroidStar
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: DroidStar
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+
+modules:
+
+  - name: imbe_vocoder
+    sources:
+      - type: git
+        url: https://github.com/nostar/imbe_vocoder.git
+        commit: 03423cb00b8e9ba1be164493e8b98b5bd7c1508c
+
+  - name: DroidStar
+    sources:
+      - type: git
+        url: https://github.com/nostar/DroidStar.git
+        commit: 2f695e63eb5a1064897198abdd65de0eaf30ca1f
+      - type: files
+        path: dudetronics.nostar.DroidStar.metainfo.xml
+      - type: files
+        path: DroidStar.desktop
+      - type: files
+        path: DroidStar.png
+    buildsystem: qmake
+    post-install:
+      - install -Dm644 DroidStar.desktop $FLATPAK_DEST/share/applications/dudetronics.nostar.DroidStar.desktop
+      - desktop-file-edit --set-key=Icon --set-value=dudetronics.nostar.DroidStar $FLATPAK_DEST/share/applications/dudetronics.nostar.DroidStar.desktop
```

Comment:
since you ship this file you should instead make the file correct in the first place.